jest.config.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  testURL: 'http://localhost',
+};
```

Comment:
what does this fix?

Reply:
Hmm, as I remember I had some problems with Jest and after googling this helped me.

Reply:
Without it I get
```
 FAIL  src/parse/__tests__/td3.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins
      
      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
          at Array.forEach (<anonymous>)
```

Reply:
could you try with `testEnvironment: "node"` instead?

Reply:
Yes, it helps.
Thanks!